charts/egressip-ipam-operator/templates/redhatcop.redhat.io_egressipams_crd.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: egressipams.redhatcop.redhat.io
+spec:
+  group: redhatcop.redhat.io
+  names:
+    kind: EgressIPAM
+    listKind: EgressIPAMList
+    plural: egressipams
+    singular: egressipam
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: EgressIPAM is the Schema for the egressipams API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: EgressIPAMSpec defines the desired state of EgressIPAM
+          properties:
+            cidrAssignments:
+              items:
+                properties:
+                  CIDR:
+                    description: TODO this is not working... kubebuilder:validation:Pattern=^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$
```

Comment:
yeah, I'd like to fix this. If you want to take a look. But please let's merge this because I have already sent the email about the release and the helm chart doesn't work. Then we can fix this. The check in is the code at the moment, so all the functionality is there anywat.